build/MUX-PR.yml

```diff
@@ -22,24 +22,24 @@ jobs:
   steps:
   - template: AzurePipelinesTemplates\MUX-BuildDevProject-Steps.yml
   
-  - template: AzurePipelinesTemplates\MUX-PublishDevProject-Steps.yml
+  - template: AzurePipelinesTemplates\MUX-PublishProjectOutput-Steps.yml
 
 # Create Nuget Package
 - template: AzurePipelinesTemplates\MUX-CreateNugetPackage-Job.yml
   parameters:
     jobName: CreateNugetPackage
     dependsOn: Build
     primaryBuildArch: x64
-  
-# Build solution that depends on nuget package
+
 - template: AzurePipelinesTemplates\MUX-NugetReleaseTest-Job.yml
   parameters:
     dependsOn: CreateNugetPackage
-    useNupkgFromArtifacts: true
+    pkgArtifactPath: '$env:artifactDownloadPath\drop'
```

Comment:
>$env:artifactDownloadPath [](start = 22, length = 25)

I thought here you used $(artifactDownloadPath) instead of the powershell $env:artifactDownloadPath? Can you make sure the PR builds are publishing the nupkg correctly still?

Reply:
> > $env:artifactDownloadPath [](start = 22, length = 25)
> 
> I thought here you used $(artifactDownloadPath) instead of the powershell $env:artifactDownloadPath? Can you make sure the PR builds are publishing the nupkg correctly still?

Hmm are we trying to make artifact path a queue-time variable too? I don't think I had $(artifactDownloadPath) before?

I just added a pkgArtifactPath parameter for MUX-NugetReleaseTest-Job.yml, the value is hardcoded to `$env:artifactDownloadPath\drop` for nuget packget and `$env:artifactDownloadPath\drop\FrameworkPackage` for framework package.

Here's an earlier PR run, looks like it added `<add key='local' value='D:\a\1\s\Artifacts\drop'/>` to nuget.config and replaced the nuget version number in proj files correctly.
https://dev.azure.com/ms/microsoft-ui-xaml/_build/results?buildId=5390&view=logs

Reply:
I see. does it work because the literal string '$env:artifactDownloadPath' is passed to powershell and evaluated by powershell?

Reply:
Yes, I believe what PowerShell saw is `<add key='local' value='$env:artifactDownloadPath\drop'/>` and then it will expand the value.

I just noticed we have $(artifactDownloadPath) defined in the NugetReleaseTest template, I've updated it to use that variable, that's probably less confusing. I did a test run it looked good.